src/test/ui/rust-2018/uniform-paths/prelude-fail-2.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2018
+
+#![feature(uniform_paths)]
+
+// Built-in attribute
+use inline as imported_inline;
+
+#[imported_inline] //~ ERROR cannot use a built-in attribute through an import
```

Comment:
Is there a test for derive helper attrs somewhere?

Reply:
Derive helper resolution is generally under-implemented and they can never be imported right now (beside other issues).

They are taken into account only when we are resolving 1) an attribute which is 2) placed on the same item with derive.